app/graphql/types/query_types/survey_poll_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  class QueryTypes::SurveyPollType < Types::BaseObject
+    field :id, ID, null: true
+    field :title, QueryTypes::I18nTextType, null: true
```

Comment:
Wieso hier der Umweg über ein (noch nicht näher definierten) neuen QueryType, wenn an der stelle doch eh ein JSON zurückkommt. 

Dann würde doch `GraphQL::Types::JSON` reichen oder?

`field :title, GraphQL::Types::JSON, null: true`

Reply:
> created I18nTextType, which is a basic JSON but with a more specific name to know about in the survey types

damit man beim lesen der types genau weiß, aaah hier gehts um einen i18n text und nicht irgendein json, wo man nicht genau weiß was sache ist

Reply:
ja, aber es ist nur ein JSON und in `pp/graphql/types/query_types/i18n_text_type.rb ` steht rein gar nichts drin, außer das es von GraphQL::Types::JSON erbt.

Reply:
Kleine der Name deute daraufhin, das es was mit Sprachen zu tun hat... auch wenn wir das I18n Modul an der stelle gar nicht verwenden

Reply:
genau, es ist ja auch nur das json type. n ur beim lesen des codes der survey types dadurch eindeutiger was für ein (unbestimmter) json genau

Reply:
also wofür entscheiden wir uns?

Reply:
```
class QueryTypes::I18nJSON < GraphQL::Types::JSON
  description <<-DESCRIPTION
    A specific JSON type representing I18n
    text data, for example:
    ```
    {
      de: \"deutscher Text\",
      en: \"englischer Text\"
    }
    ```
  DESCRIPTION
end
```